.travis.yml

```diff
@@ -11,7 +11,7 @@ before_install:
   - sudo sh -c 'PATH=/usr/local/node/bin:$PATH yarn global add serve'
 install:
   - make
-  - make install
+  - make dev-install
```

Comment:
Why should we modify this line?

Reply:
Thanks, the change is removed.

Reply:
make install and make easy-install do not work here,  change back to make dev-install